src/DataDefinitionsBundle/Migrations/Version20190731104917.php

```diff
@@ -13,6 +13,11 @@ class Version20190731104917 extends AbstractPimcoreMigration
     public function up(Schema $schema)
     {
         if ($schema->hasTable('import_definitions_log')) {
+            // delete just created table before renaming the old table
+            if ($schema->hasTable('data_definitions_import_log')) {
```

Comment:
If the table exists, we should skip the migrations, and not delete it right?

Reply:
The new table (`data_definitions_import_log`) is being created during the bundle installation [see here](https://github.com/w-vision/DataDefinitions/blob/0d966da0a6bf1568e1a54e582d696a4327f6a514/src/DataDefinitionsBundle/DependencyInjection/Configuration.php#L128). The problem is if you used the bundle then this table already exists as `import_definitions_log` so it has to be renamed. But it could not be renamed because the `data_definitions_import_log` table already exists too. 
We have two ways of solving this issue:
1. Do not create `data_definitions_import_log` during installation if table `import_definitions_log` exists.
2. Delete the `data_definitions_import_log` table and rename the `import_definitions_log` table into `data_definitions_import_log`. 
I chose the second way in order to keep [the bundle configuration file](https://github.com/w-vision/DataDefinitions/blob/master/src/DataDefinitionsBundle/DependencyInjection/Configuration.php) simple and put all the logic related to migration from the older version to the new one in one place

Reply:
ok, alright, you are right. Lets delete it, since the `import_definitions_log` must be the accurate one for these cases